spec/javascripts/models/beta/story_spec.js

```diff
@@ -210,4 +210,79 @@ describe('Story model', function() {
       });
     });
   });
-})
+
+  describe("toggleStory", () => {
+    describe("When story is collapsed", () => {
+      it("expand story", () => {
+        const story = { collapsed: true }
+
+        const expandedStory = Story.toggleStory(story);
+
+        expect(expandedStory.collapsed).toEqual(false);
+      });
+    });
+
+    describe("When story is expanded", () => {
+      it("collapse story", () => {
+        const story = { collapsed: false }
+
+        const collapsedStory = Story.toggleStory(story);
+
+        expect(collapsedStory.collapsed).toEqual(true);
```

Comment:
Can we check the whole content of `collapsedStory` so we know `toggleStory` doesn't change more than the value of `collapsed`?